static/app/views/performance/transactionSummary/transactionReplays/content.tsx

```diff
@@ -76,7 +77,7 @@ function ReplaysContent({
         isFetching={isFetching}
         replays={replays}
         showProjectColumn={false}
-        sort={eventView.sorts[0]}
+        sort={first(eventView.sorts) || {field: 'createdAt', kind: 'asc'}}
```

Comment:
Using `[0]` was unsafe, it could return `undefined` if the array is empty.

Reply:
I like to avoid lodash too; but that's not 100% the same it's more like 'if the var is not an array' and the problem is that i have an array, but the length could be zero. `noUncheckedIndexedAccess` seems like the compilerOption, maybe we can enable that separately actually.

https://www.typescriptlang.org/tsconfig#noUncheckedIndexedAccess

Reply:
Ah right -- assuming we had that rule, would adding the default value appease the compiler? Or would we need an explicit length check?

Reply:
with/without the rule the types are:
```
const a: string[] = [];

// Without rule:
const first = a[0]; // type == string

// With rule:
const first = a[0]; // type == string | undefined
```